.gitignore

```diff
@@ -0,0 +1,3 @@
+*.class
+*.iml
+.idea
```

Comment:
QQ: Should we need this?

Reply:
To ignore IDE (IntelliJ) specific settings